src/styles/blocks/card/card.scss

```diff
@@ -0,0 +1,86 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: $card-width;
+  padding: 32px 16px 16px;
+  color: $main-font-color;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $secondary-font-color;
+    margin-bottom: 16px;
+  }
+
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    font-family: 400;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+    line-height: 18px;
+  }
+
+  &__price-title {
+    font-size: 12px;
+    color: $secondary-font-color;
+    font-weight: 400;
+  }
+
+  &__price-sum {
+    font-size: 16;
```

Comment:
Missing unit for font-size, it should be specified in px or another suitable unit.

Reply:
+